src/propsheet/console.cpp

```diff
@@ -115,7 +115,7 @@ void SaveConsoleSettingsIfNeeded(const HWND hwnd)
                 LoadStringW(ghInstance, IDS_LINKERROR, awchBuffer, ARRAYSIZE(awchBuffer));
                 StringCchPrintf(szMessage,
                                 ARRAYSIZE(szMessage),
-                                awchBuffer,
+                                awchBuffer, // CodeQL [SM01734] Pulled from a resource file and cannot be a string literal

```

Comment:
wait, didn't you fix one of these already? Does CodeQL spit out a single error every time? Can it be made to report all of them instead so we can fix them all in one PR?

Reply:
@DHowett Yeah I didn't realize it wasn't one of the files under OSS that got fixed for 1.22. This is the last one we got with our set of 3 from a couple months ago. I don't expect any more of these